examples/energy_mng/src/main.rs

```diff
@@ -24,7 +30,13 @@ pub fn main() {
     #[cfg(all(debug_assertions, target_arch = "wasm32"))]
     console_error_panic_hook::set_once();
 
-    MainWindow::new().unwrap().run().unwrap()
+    let window = MainWindow::new().unwrap();
+
+    let header_timer = header::setup(&window);
+    weather::setup(&window);
+
+    window.run().unwrap();
+    header_timer.stop()
```

Comment:
That's not needed.

Reply:
Do you mean `header_timer.stop()`? But I need to store the timer right? Without it the timer is dropped.

Reply:
correct.  thinking of it i think it make sense like that.  
You could just call it `_header_timer`, but this makes it more explicit like you did. I like that.